package-lock.json

```diff
@@ -2479,7 +2479,6 @@
 				"@wordpress/compose": "file:packages/compose",
 				"@wordpress/core-data": "file:packages/core-data",
 				"@wordpress/data": "file:packages/data",
-				"@wordpress/deprecated": "file:packages/deprecated",
```

Comment:
😱 is that correct?

Reply:
(I think we added a deprecation in #11543, fwiw.)

Reply:
It is correct in that the current state of master after this branch would have no occurrence of `@wordpress/deprecated` in `edit-post`.

Reply:
It's correct as the `edit-post` package don't have any other deprecations anymore.

Ideally, we don't add more deprecations now as we're in API Freeze and keep supporting both APIs unless we think it's very unlikely that someone is using the said API in which I case you can add the dependency back in the PR.

Reply:
But we could have API changes post-5.0, right? I assumed we'd still want to be able to deprecate APIs after the core merge.

Reply:
@tofumatt yes, but the process is not defined yet.

Reply:
The dependencies should only be those which are actively used. If we need to deprecate in the future, it can be brought back.